flow/flow.proto

```diff
@@ -183,6 +189,11 @@ message Flow {
   repeated RawPacket LastRawPackets = 36;
 /* number of raw packet captured */
   int64 RawPacketsCaptured = 37;
+
+/* counts the number of times this flow was reported */
+  int64 ReportCounter = 60;
```

Comment:
I wouldn't keep this. If we want to know how many time a flow has been reported we can check the `_version` field of elasticsearch or even count the number of metric records. It seems to be more a debugging information ?

Reply:
haha
https://github.com/skydive-project/skydive/pull/1532#discussion_r242832475
But OK, we'll ask @eranra about this